docs/userguide/networking/work-with-networks.md

```diff
@@ -0,0 +1,454 @@
+relied on. In addition to this network, you can create your own `bridge` or `overlay` network.  
+
+A `bridge` network resides on a single host running an instance of Docker Engine.  An `overlay` network can span multiple hosts running their own engines. If you run `docker network creat`e and supply only a network name, it creates a bridge network for you.
+
+```bash
+$ docker network create simple-network
+de792b8258895cf5dc3b43835e9d61a9803500b991654dacb1f4f0546b1c88f8
+$ docker network inspect simple-network
+[
+    {
+        "Name": "simple-network",
+        "Id": "de792b8258895cf5dc3b43835e9d61a9803500b991654dacb1f4f0546b1c88f8",
+        "Scope": "local",
+        "Driver": "bridge",
+        "IPAM": {
+            "Driver": "default",
+            "Config": [
+                {}
+            ]
+        },
+        "Containers": {},
+        "Options": {}
+    }
+]
+```
+
+Unlike `bridge` networks `overlay` networks require some pre-existing conditions
+before you can create one. These conditions are:
+
+* Access to a key-value store. Engine supports Consul Etcd, and Zookeeper (Distributed store) key-value stores.
+* A cluster of hosts with connectivity to the key-value store.
+* A properly configured Engine `daemon` on each host in the cluster.
+
+The `docker daemon` options that support the `overlay` network are:
+
+* `--cluster-store`
+* `--cluster-store-opt`
+* `--cluster-advertise`
+
+It is also a good idea, though not required, that you install Docker Swarm on
+to manage the cluster. Swarm provides sophisticated discovery and server
+management that can assist your implementation.
+
+When you create a network, Engine creates a non-overlapping subnetwork for the network by default. You can override this default and specify a subnetwork directly using the the `--subnet` option. On a `bridge` network you can only create a single subnet. An `overlay` network supports multiple subnets.
+
+In addition to the `--subnetwork` option, you also specify the `--gateway` `--ip-range` and `--aux-address` options.
+
+```bash
+docker network create -d overlay
+  --subnet=192.168.0.0/16 --subnet=192.170.0.0/16
+  --gateway=192.168.0.100 --gateway=192.170.0.100
+  --ip-range=192.168.1.0/24
+  --aux-address a=192.168.1.5 --aux-address b=192.168.1.6
+  --aux-address a=192.170.1.5 --aux-address b=192.170.1.6
+  my-multihost-newtork
+```
+
+Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
+
+## Connect containers
+
+Docker containers can dynamically connect to one or more networks. These
+networks can be backed the same or different network drivers. Once connected,
+the containers can communicate using only another container's IP address or
+name.
+
+For `overlay` networks or custom plugins that support multi-host
+connectivity, containers connected to the same multi-host network but launched
+from different Engines can also communicate in this way.
```

Comment:
I'd change "from different Engines" to "on different hosts" (I'd confuse different Engines with, e,g, "Docker" and "rkt" ;-))